qemu/tests/block_hotplug.py

```diff
@@ -121,39 +123,43 @@ def get_new_disk(disk1, disk2):
 
             device_list.append(device)
             if not new_disks:
-                raise exceptions.TestFail("Cannot find new disk after hotplug.")
+                test.fail("Cannot find new disk after hotplug.")
 
             if params.get("need_plug") == "yes":
                 disk = new_disks[0]
             else:
                 disk = new_disks[num]
 
+            session = vm.wait_for_login(timeout=timeout)
+            if params.get("format_disk", "no") == "yes":
+                if i == 0:
```

Comment:
please not use single character.

Reply:
Hi coli,

Got your meaning, just this character "i" is come from the main cycle:  
for i in xrange(repeat_times):
        error_context.context("Hotplug block device (iteration %d)" % i,
                              logging.info)
        .............
            if params.get("format_disk", "no") == "yes":
￼                  if i == 0: 
"if i == 0" used to limit the time of format_disk, after the first time hotplug done, need to execute the format_disk, after other hotpluged, no need to run format_disk.

Reply:
yes, I know it's inherited from the original code, but could you please help update it?

Thanks.

Reply:
that's ok, do you think about changing to "time"? or "repeat_time"? or any better advise? I just think "name == 0" no better than "i == 0"~